### ISSUE TITLE
Regarding the selection of hyperparameters

### ISSUE THREAD

**Issue by lscpku** [P2333/Adaptive-Diversity-Promoting#3]
Hi, thanks for releasing the code. I have read your paper and have some questions regarding the selection of hyperparameter alpha. In Section 4.2, it says alpha=2 is chosen according to Eq. (7). However, by my calculation, let K=3 and F_y=0.9, alpha should be around 0.76 when L=10 and around 0.49 when L=100. Also it seems that the paper did not mention why beta is chosen as 0.5. Could you please explain how the hyperparameters alpha and beta are selected in your experiments? Thanks.

**Comment by P2333** [comment]
Thank you for the questions.

About the choice of \alpha, Eq.(7) gives an instruction, but another way is to choose a certain \alpha for training, and then scale the logits with a constant temperature to calibrate the predictions in inference. The \alpha only needs to guarantee that the non-maximal predictions have enough numerical significance during training.

About the choice of \beta, we do not provide a formal instruction, so the selection is based on cross-validation.

**Comment by lscpku** [comment]
Thanks for the clarification!
